x-pack/plugins/observability_solution/infra/server/lib/alerting/metric_threshold/register_metric_threshold_rule_type.ts

```diff
@@ -63,14 +52,15 @@ export function registerMetricThresholdRuleType(
   if (!featureFlags.metricThresholdAlertRuleEnabled) {
     return;
   }
-
+  const comparators: string[] = Object.values(COMPARATORS);
+  comparators.push(LEGACY_OUTSIDE_RANGE);
   const baseCriterion = {
     threshold: schema.arrayOf(schema.number()),
-    comparator: oneOfLiterals(Object.values(Comparator)),
+    comparator: oneOfLiterals(comparators),
     timeUnit: schema.string(),
     timeSize: schema.number(),
     warningThreshold: schema.maybe(schema.arrayOf(schema.number())),
-    warningComparator: schema.maybe(oneOfLiterals(Object.values(Comparator))),
+    warningComparator: schema.maybe(oneOfLiterals(Object.values(COMPARATORS))),
```

Comment:
Seems we have a similar situation for warningComparator as well

![image](https://github.com/elastic/kibana/assets/12370520/0e103702-5719-48c5-9abe-6e391f89f7d4)

Reply:
Did you create a rule in the main and then change it to your branch? Last time, I tested it like this and got the above error in your branch.

Reply:
@maryam-saeidi, good catch. Fixed!